### ISSUE TITLE
Feature request: send role cards

### ISSUE THREAD

**Issue by yamikuronue** [yamikuronue/sockMafia#81]
There should be a simple command, something like !send <user> <text> so you can send role cards from the bot and walk away.

**Comment by AccaliaDeElementia** [comment]
hmm.... well i created a script to do that from console so it should be possible to make the bot do that too...

how are you going to get the bot to read the content of the user cards?

**Comment by yamikuronue** [comment]
The easy solution is to do something like

```
!send Accalia You are the **Godfather**, you have a night kill and are also immune to all forms of death, you win all the time because town sucks
```

and it would send that role card. So you could then just paste from your prep sheet.

The harder solution would be to allow a template definition or three so you could do 

```
!send Accalia role Godfather
!send Accalia scumText
!send Accalia scumWinCon
```

But I think I'll make that a separate issue

**Comment by yamikuronue** [comment]
Ooh, another thought: if you can set them in a way taht persists, and have them echo'd back for proofreading purposes, you can then have them all sent at once with a single !send all

**Comment by AccaliaDeElementia** [comment]
hmm... guess i should alter the DAO to let you store arbitrary data on the users as well as on the Game?

**Comment by AccaliaDeElementia** [comment]
Resolved by #136
